src/bcoinWallet.ts

```diff
@@ -172,6 +174,34 @@ export class TestnetBitcoinWallet implements BitcoinWallet {
         return "150";
     }
 
+    public async showProgressBar() {
+        let progress = await this.node.chain.getProgress();
+        let walletState = await this.walletdb.getState();
+        const b1 = new cliProgress.SingleBar({
+            format: 'CLI Progress |' + '{bar}' + '| {percentage}% || {value}/{total} Chunks || Known Height: {wallet}/{height}',
+            barCompleteChar: '\u2588',
+            barIncompleteChar: '\u2591',
+            hideCursor: true
+        });
+
+        b1.start(100, progress * 100, {
+            wallet: walletState.height,
+            height: this.node.chain.height,
+        });
+
+        while (progress < 1) {
+            await sleep(30000);
```

Comment:
```suggestion
            await sleep(3000);
```
`30000` is a bit long I feel, that's half a minute. When I first ran it I was like "is it actually doing something"? Is it a typo and supposed to be `3000`? 
(Does not hurt to update the progress bar in shorter intervals...)